doc/manuals/user/cors.md

```diff
@@ -22,7 +25,7 @@ maximum preflight cache time
 
         contextBroker -corsOrigin specificdomain.com
 
-CORS is available for all /v2 resources but for /v1 resources, it is only

```

Comment:
I see all in **bold** from this point ... A "bold-end" missing somewhere?

Reply:
It could be some "crazy diff effect" in github. The view buttom for that file in the branch (https://github.com/telefonicaid/fiware-orion/blob/hardening/cors_doc/doc/manuals/user/cors.md) shows it is fine.